src/localization/localization.js

```diff
@@ -1,12 +1,52 @@
-import englishMessages from './../locales/en.json';
+import CzechMessages from './../locales/cs.json';
+import GermanMessages from './../locales/de.json';
+import EnglishMessages from './../locales/en.json';
+import SpanishMessages from './../locales/es.json';
+import FrenchMessages from './../locales/fr.json';
+import ItalianMessages from './../locales/it.json';
+import JapaneseMessages from './../locales/ja.json';
+import KoreanMessages from './../locales/ko.json';
+import PolishMessages from './../locales/pl.json';
+import PortugueseBrazilMessages from './../locales/pt-BR.json';
+import RussianMessages from './../locales/ru.json';
+import SimplifiedChineseMessages from './../locales/zh-Hans.json';
+import TraditionalChineseMessages from './../locales/zh-Hant.json';
 
 export const getMessagesForLocale = (locale) => {
   switch (locale) {
   case 'en':
-    return englishMessages;
+    return EnglishMessages;
```

Comment:
How about shorter variable names like `enus`,`eses` etc.?

Reply:
I thought these are passed by Dynamo so only the longer ones are passed